resources/assets/components/Users/UserInformation.js

```diff
@@ -14,7 +14,16 @@ const UserInformation = props => (
         }
         <p>
           {props.user.email ? <span>{props.user.email}<br /></span> : null}
-          {props.user.mobile ? <span>{props.user.mobile}<br /></span> : null }
+
+          {props.user.mobile ?
+              {props.linkSignup && isEmpty(props.user.first_name) ?
+                <span><a href={`/signups/${props.linkSignup}`}>{props.user.mobile}</a><br /></span>;
+                :
+                <span>{props.user.mobile}<br /></span>;
+              }
+            : null
+          }
+
           {displayCityState(props.user.addr_city, props.user.addr_state) ?
```

Comment:
@sbsmith86 this is giving me an error when I do the nested ternary. I think this is ok to do but when I don't do a nested ternary it works. Is this because of PropTypes rules perhaps?

Reply:
It's an `eslint` thing. Is the `no-nested-ternary` rule. So you probably need an actual if-statement here instead.

Reply:
ah I see - thanks for clarifying @sbsmith86 !